### PR TITLE
Creating a NuGet package for libuv darwin

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -46,10 +107,80 @@ var FULL_VERSION = '${PRODUCT_VERSION + "-" + E("DNX_BUILD_VERSION")}'
...
             var p = (int)Environment.OSVersion.Platform;
             return (p != 4) && (p != 6) && (p != 128);
         }
     }
+
+    bool CanBuildForDarwin
+    {
+        get
+        {
+            return string.Equals(Uname(), "Darwin");
```
Comment (muratg) [inline makefile.shade]: I assume not, but do we need case insensitivity here?

Comment (moozzyk) [inline makefile.shade]: No, uname returns "Darwin" (also these helper functions were mostly copied from dnx)

Comment (muratg) [inline makefile.shade]: :+1:

**Comment by muratg** [comment]
:shipit:

**Comment by moozzyk** [comment]
7b8f10a89f01d8171104b9fd15a0a6752388ab2f

**Pull request closed without merging**